### PR TITLE
Add is_type command, is_instance_of command

### PR DESCRIPTION
**Pull request by xVin**
targets #492

### CODE REVIEW

**Comment by neithernut** [comment]
Mh... I think this command should be renamed into `is_instance_of` or something, as you need two objects for this operation. You can perform a typecheck usng a type-_string_ using `ws_object_has_type`. If you want to, you cann add such a command in this PR.

In the end, this command is not what was requested, but we may need it either way, so this is actually useful.

**Comment by xVin** [comment]
I must have gotten confused somewhere along the way o.o
I will add the is_type command asap.

**Comment by xVin** [comment]
No, wait. I did this because the hastypename command already exists and I wondered what else the is_type command could be supposed to do then.

Edit: I renamed this command anyway, but I guess I need some clarification on what is_type should be doing. Re-reading the issue, it should do typechecking for values only?

**Comment by neithernut** [comment]
I think we just forgot to close that issue...
Sorry...

**Comment by xVin** [comment]
Sure? Because the hastypename command already existed when the issue was opened.

**Comment by neithernut** [comment]
Er.... year, I got confused here...
We _do_ have a command to check an object's typename via API. What's actually missing is a command for checking the type of a `ws_value` itself.

**Comment by xVin** [comment]
Ok so I'll just add that now :)

File: src/command/object.c [hunk 1]
```diff
@@ -151,3 +151,26 @@ ws_builtin_cmd_has_attr(
...
+ws_builtin_cmd_is_instance_of(
+    union ws_value_union* args
+) {
+    if ((args[0].value.type != WS_VALUE_TYPE_OBJECT_ID) ||
+        (args[1].value.type != WS_VALUE_TYPE_OBJECT_ID)) {
+        return -EINVAL;
+    }
+
+    struct ws_object* obj = ws_value_object_id_get(&args[0].object_id);
+    ws_object_type_id* id = ws_value_object_id_get(&args[1].object_id)->id;
```
Comment (matthiasbeyer) [inline src/command/object.c:170]: I'm not sure, but I think this function could possibly return `NULL`, resulting in a segfault here. Check the return value please!

**Comment by matthiasbeyer** [comment]
Besides the one annotation this looks good!

ACK.

**Comment by xVin** [comment]
Update.

**Comment by neithernut** [comment]
ACK.

**Pull request merged**